.travis.yml

```diff
@@ -1,12 +1,23 @@
 sudo: false
 language: node_js
 node_js:
+  - "4.1"
+  - "4.0"
```

Comment:
We don't have to give both the above versions. Please just add `4` and it should be enough.

Also, please cherry pick this commit and use it before your commits in this PR. https://github.com/apiaryio/protagonist/commit/15b7c9d296f21e701cb7ab737a0c80ff90c37295